apps/documentation/src/routes/documentation/cli.mdx

```diff
@@ -21,6 +19,22 @@ Tuono is the CLI that provides all the needed commands to handle the full-stack
 
 To list all the available commands, run `tuono -h`.
 
+```bash
+The react/rust fullstack framework
+
+Usage: tuono <COMMAND>
+
+Commands:
+  dev    Start the development environment
+  build  Build the production assets
+  new    Scaffold a new project
+  help   Print this message or the help of the given subcommand(s)
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible typo: you repeated a whitespace (WHITESPACE_RULE)
Suggestions: ` `
Rule: https://community.languagetool.org/rule/show/WHITESPACE_RULE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChA5MjI5N2VhMTJhZmM2OTYxEgxMYW5ndWFnZVRvb2w= -->